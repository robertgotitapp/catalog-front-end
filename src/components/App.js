import { connect } from 'react-redux';
import { withRouter } from 'react-router';
import Main from './Main';

export const App = withRouter(connect()(Main));

export default App;
