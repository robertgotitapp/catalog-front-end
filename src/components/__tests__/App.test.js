import React from 'react';
import { shallow } from 'enzyme';
import { App } from '../App';

describe('components/Home', () => {
  const container = shallow(<App />);

  it('should render correctly', () => {
    expect(container).toMatchSnapshot();
  });
});
