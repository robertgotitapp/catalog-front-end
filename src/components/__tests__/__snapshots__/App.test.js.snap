// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Home should render correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
